src/error/record.rs

```diff
@@ -38,17 +38,23 @@ impl fmt::Display for RecordError {
 
 #[derive(Error, Debug)]
 pub enum AliasConversionError {
-    #[error("alias must not contain a colon")]
-    IsRemoteId,
-    #[error("alias must be non-empty")]
-    Empty,
+    #[error("Invalid alias '{0}': alias must not contain a colon")]
+    IsRemoteId(String),
+    #[error("Invalid alias '{0}': alias must be non-empty")]
+    Empty(String),
```

Comment:
The reason initially for not including this message initially to suppress the weird warning when `Alias` is read from the command line.

Reply:
e.g. the error message which occurs writing something like `autobib alias add '::' <something>`

Reply:
But overall, I think this handling of errors is better. Is there a way to special-case the CLI handling somehow?

Reply:
We could defer the type conversion, i.e. accept any string and convert manually.

Reply:
Sounds good to me!